src/SparkFun_Ublox_Arduino_Library.cpp

```diff
@@ -1029,6 +1029,7 @@ void SFE_UBLOX_GPS::processUBXpacket(ubxPacket *msg)
       gnssFixOk = extractByte(21 - startingSpot) & 0x1; //Get the 1st bit
       diffSoln = extractByte(21 - startingSpot) >> 1 & 0x1; //Get the 2nd bit
       carrierSolution = extractByte(21 - startingSpot) >> 6; //Get 6th&7th bits of this byte
+      headVehValid = extractByte(21 - startingSpot) >> 5 & 0x1; // Get the 5th bit
```

Comment:
Let's change this to:
```headVehValid = (extractByte(21 - startingSpot) >> 5) & 0x1; // Get the 5th bit```
just to make it clear that the shift happens before the AND

Reply:
Actually, can you change line 1030 to please?
(I know the code works - these changes just help with the clarity)

Reply:
Sure. I will update accordingly. Thanks.